plexapi/audio.py

```diff
@@ -527,6 +529,22 @@ def metadataDirectory(self):
         guid_hash = utils.sha1hash(self.parentGuid)
         return str(Path('Metadata') / 'Albums' / guid_hash[0] / f'{guid_hash[1:]}.bundle')
 
+    def sonicAdventure(
+        self: TAudio,
+        to: TAudio,
+        **kwargs: Any,
+    ) -> list[TAudio]:
+        """Returns a sonic adventure from the current track to the specified track.
+
+        Parameters:
+            to: The target track for the sonic adventure.
+            **kwargs: Additional options passed into :func:`~plexapi.library.MusicSection.sonicAdventure`.
```

Comment:
Add types to doc string.

Reply:
yes, upper bound should be Track but Audio works too since `self` will always be a Track and `to` will expect it to be of the same class